dracut/30ignition/coreos-remove-initramfs-network.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/bash
+# -*- mode: shell-script; indent-tabs-mode: nil; sh-basic-offset: 4; -*-
+# ex: ts=8 sw=4 sts=4 et filetype=sh
+
+# Clean up the interfaces set up in the initramfs
+for f in /sys/class/net/*; do
+    interface=$(basename "$f")
+    ip link set $interface down
```

Comment:
Is this enough to kill `dhclient`?  I think we also want to loop over `/run/dhclient*.pid` or so, actually before doing the interfaces.

Reply:
dhclient isn't actually running AFAICT in the initramfs (nothing from ps aux). It doesn't try to re-lease if I down the interfaces at least.